src/util/scrollbar.ts

```diff
@@ -0,0 +1,44 @@
+import {Injectable, Inject} from '@angular/core';
+import {DOCUMENT} from '@angular/common';
+
+
+
+@Injectable()
+export class ScrollBar {
```

Comment:
Could you add a bit of JS doc to this class with a brief description on how it works, what it can be used for etc.?

Reply:
Yes, since I finally made it as a separate utility it makes sense